web/package.json

```diff
@@ -36,6 +36,10 @@
   },
   "homepage": "https://github.com/18F/cms-hitech-apd#readme",
   "dependencies": {
+    "@fortawesome/fontawesome": "^1.1.4",
```

Comment:
I was a little worried about what FA does to a11y, but it turns out they're handling that internally, and this React wrapper doesn't interfere with it!  :tada:  Checked it locally to make sure it was rendering properly and everything looks gravy.

Reply:
i thought they might, given its popularity